bgpd/bgp_route.c

```diff
@@ -13161,25 +13146,25 @@ static void show_adj_route(struct vty *vty, struct peer *peer, afi_t afi,
 					continue;
 				}
 
-				if (type == bgp_show_adj_route_received &&
-					(route_filtered || ret == RMAP_DENY))
-					filtered_count++;
+				if (type == bgp_show_adj_route_received
+				    && (route_filtered || ret == RMAP_DENY))
+					*filtered_count += 1;
```

Comment:
I would prefer `*filtered_count++;`

Reply:
`*filtered_count++` doesn't work. It increments the pointer address instead of the value stored at the memory address. I figured it was either `filtered_count += 1` or `(*filtered_count)++` and I thought this was cleaner to read.

Reply:
Yes, sorry, I was about (*filtered_count)++.

Reply:
I can update the syntax to use that method. I just made a judgement call on what was subjectively easier to read :-)

Reply:
Updated to use (*filtered_count)++ syntax